apps/ledger-live-mobile/android/app/src/main/java/com/ledger/live/newarchitecture/components/MainComponentsRegistry.java

```diff
@@ -0,0 +1,31 @@
+package com.ledger.live.newarchitecture.components;
```

Comment:
This file isn't used yet as we are not activating the new RN architecture (Fabric + TurboModules)